src/ExtensionSet.php

```diff
@@ -191,13 +170,9 @@ public function getFunctions()
     }
 
     /**
-     * Get a function by name.
-     *
-     * @param string $name function name
-     *
      * @return TwigFunction|false
```

Comment:
it would probably make sense to change that to `TwigFunction|null` in Twig 3 (or maybe even in current versions in case this is internal code)

Reply:
That's internal code, but I prefer to change it in 3.0.